### PR TITLE
docker-compose-completion: deprecate

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by branchvincent** [comment]
since v2, this no longer has completions (see https://github.com/docker/compose/issues/8550). i'm not sure what to do with this formula

**Comment by chenrui333** [comment]
oh, then we should deprecate this formula. let me update the PR.

File: Formula/docker-compose-completion.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class DockerComposeCompletion < Formula
   desc "Completion script for docker-compose"
   homepage "https://docs.docker.com/compose/completion/"
-  url "https://github.com/docker/compose/archive/1.29.2.tar.gz"
-  sha256 "99a9b91d476062d280c889ae4e9993d7dd6a186327bafb2bb39521f9351b96eb"
+  url "https://github.com/docker/compose/archive/v2.0.1.tar.gz"
+  sha256 "5a1b1fdb9681c6f4b39fceab7d7dca12285e72cb55e3d35c31f4659cc939cd2b"
```
Comment (branchvincent) [inline Formula/docker-compose-completion.rb:5]: assuming we should revert this?

Comment (branchvincent) [reply Formula/docker-compose-completion.rb:5]: also, do we need to remove the livecheck block?

Comment (SMillerDev) [reply Formula/docker-compose-completion.rb:5]: Yes and yes

**Review by branchv — APPROVED** [review]

File: Formula/docker-compose-completion.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/docker-compose-completion.rb]: > assuming we should revert this?

That is right, Thanks!!

**Pull request closed without merging**